src/registrar/templates/base.html

```diff
@@ -159,7 +159,7 @@
 
   {% block wrapper %}
   {% block wrapperdiv %}
-  <div id="wrapper">
+  <div id="wrapper" class="wrapper--padding-top-6">
```

Comment:
Old code, but why not padding-top-6?

Reply:
Padding-top-6 gets overridden by wrapper css